packages/ui-kit/src/components/content-notifications.tsx

```diff
@@ -10,7 +9,7 @@ import {
 import { TypographyContainer } from './markdown';
 import { colors, tokens, dimensions } from '../design-system';
 
-const getIconByType = (type) => {
+const getIconByType = (type: string) => {
```

Comment:
Better to use the props type:

```suggestion
const getIconByType = (type: ContentNotificationProps['type']) => {
```